ArduCopter/system.cpp

```diff
@@ -79,6 +79,9 @@ void Copter::init_ardupilot()
 
     // initialise notify system
     notify.init();
+
+    //Setting the mode to INITIALISING
+    set_mode(INITIALISING, MODE_REASON_UNKNOWN);
```

Comment:
Add a better entry to the enumeration.

`MODE_REASON_INITIALISING` would seem to be reasonable....

Reply:
I should also add - please add the entry to the end of that enumeration.  While it doesn't have assigned values in the enumeration, we do log these values so stability is a good thing.

Reply:
MODE_REASON_ENUMERATION added